CONTRIBUTING.md

```diff
@@ -0,0 +1,129 @@
+##Contributing to Tasties project
+
+---
+
+Hello and welcome to Tasties project. 
+Please reading and following these guidelines will help us make the contribution process easy and effective for everyone involved.
+
+
+
+### Quicklinks
+
+---
+
+- [Pre-Requirements](#Pre-Requirements)
+- [Issues](#Issues)
+- [Pull Requests](#Pull Requests)
+- [Testing](#Testing)
+- [Code of Conduct](#Code of Conduct)
+- [Getting Help](#Getting Help)
+
+
+### Pre-Requirements
+
+---
+
+To contribution to our project please download and install the following:
+
+- Vagrant.
+- VirtualBox(or other VM that works with "Vagrant").
+
+After the installations typing 'vagrant up' and automatically the development environment will set up.
+
+
+### Issues
+
+---
+
+Issues should be used to report problems with the library, request a new feature, or to discuss potential changes before a PR is created.
+When you open issues use there [words](#https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests).Issues can also help the team know what every member of the team work on.
```

Comment:
These keywords should be used in the PR to link an issue, and not in the PR itself